### PR TITLE
Various Hover Tweaks

### PR DESCRIPTION
**Pull request by natyusha**
- remove hover from disabled buttons
- allow button tooltip placement to be defined with a fallback to "top"
- move theme's logo css below variables from the tailwind config
- add hover to series page tabs

### CODE REVIEW

File: src/components/Input/Button.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import type { PlacesType } from 'react-tooltip';
```
Comment (harshithmohan) [inline src/components/Input/Button.tsx:2]: Can this be moved to below line 7 (with a one line gap after 7)?
If linter complains, move it back 😔

File: src/components/Input/Button.tsx [hunk 2]
```diff
@@ -14,6 +15,7 @@ type Props = {
   loading?: boolean;
   loadingSize?: number;
   onClick?: React.MouseEventHandler<HTMLButtonElement>;
+  place?: PlacesType;
```
Comment (harshithmohan) [inline src/components/Input/Button.tsx:18]: Rename to `tooltip-place`

**Review by harshithmohan — CHANGES_REQUESTED** [review]

File: src/components/Input/Button.tsx (hunk 1 above)
Comment (natyusha) [reply src/components/Input/Button.tsx:2]: Unfortunately: `react-tooltip type import should occur before import of @mdi/js  import/order`

File: src/components/Input/Button.tsx (hunk 2 above)
Comment (natyusha) [reply src/components/Input/Button.tsx:18]: Used `tooltipPlace` to avoid a hyphen in the property names, should be close enough I think

**Pull request merged**